spk/minio/src/wizard/upgrade_uifile.sh

```diff
@@ -114,6 +114,23 @@ cat <<EOF > $SYNOPKG_TEMP_LOGFILE
             ]
          }
       ]
+   },
+   {
+      "step_title": "MinIO - end of life for filesystem mode",
+      "items": [
+         {
+            "desc": "MinIO removed support for the filesystem mode which is a 1:1 mapping of buckets to files on disk. If your deployment uses filesystem mode you need to manually migrate to single-node single-drive deployment setup. This is the only way to receive future updates of MinIO. A new MinIO package with a different name will be provided in future."
+         },
+         {
+            "desc": "To check if you're affected by this change check your buckets manually by looking into each folder on your diskstation. If there is exactly a single file per S3 object you need to migrate your data. Otherwise you're already running a single-node single-drive deployment an can upgrade to the new MinIO package in future without a migration."
+         },
+         {
+            "desc": "Because of the complexity and risks involved an automatic upgrade process is not available. The official MinIO documentation provides a guide for doing so called 'Migrate from Gateway or Filesystem Mode'"
+         },
+         {
+            "desc": "For a regular user it is suggested to run an instance of MinIO on a seperate system and migrate your data to this new instance. This system could be a desktop, single board computer or a server. If possible you can mount the storage from you diskstation over the network and use it directly during the migration process. After a successful migration replace the data of your current MinIO installation on your diskstation. Please stop MinIO before replacing your data and restart it afterwards."
```

Comment:
```suggestion
            "desc": "For a regular user it is suggested to run an instance of MinIO on a seperate system and migrate your data to this new instance. This system could be a desktop, single board computer or a server. If possible you can mount the storage from your diskstation over the network and use it directly during the migration process. After a successful migration replace the data of your current MinIO installation on your diskstation. Please stop MinIO before replacing your data and restart it afterwards."
```

Reply:
fixed in https://github.com/SynoCommunity/spksrc/pull/5555/commits/6ecaa703bce6800a749d049e4c1d9a2c5091eefe